limitador/CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 Notable changes to Limitador will be tracked in this document.
 
+## 0.3.0 - 2022-12-20
+
+- Added Infinispan limits storage (first version) #38 #43 #44 #45 
+- Perform semver-compatible dependency updates and move to edition 2021 #51
+- Update Limitador to Tokio 1 async reactor #54
```

Comment:
`#54` is not link-able. I guess, it should be `[#54](https://github.com/Kuadrant/limitador/pull/54)`

Reply:
Oh, yeah... stolen from the release 🤦 I'll update them all